Druid/DRCogitCanonicaliser.class.st

```diff
@@ -60,6 +64,14 @@ DRCogitCanonicaliser >> collapseBitwiseOperationIfPossible: aDRGetConditionCode
 	^ newInstruction
 ]
 
+{ #category : 'initialization' }
+DRCogitCanonicaliser >> initialize [
+	collapsibleLogicAndBitwiseInstructions := Dictionary new
+		at: DRBitAnd put: { DRNoTestMask. DRTestMask };
+		at: DRAnd put: { DRTestNand. DRTestAnd };
```

Comment:
Why not `DRNoTestAnd`? To be consistent with the mask. Also _nand_ and _and_ are more similar and could be miss-read.